doc/manual/plugins_global/rc.rst

```diff
@@ -110,3 +110,72 @@ usually interpreted as a program option.  In order to pass a signed
 integer you may need to do something like::
 
     $ grc -- channel FOO zoom -7
+
+
+Interfacing from within Python
+==============================
+
+It is also possible to control Ginga in RC mode
+from within Python.   The following describes
+some of the functionality.
+
+Connecting
+----------
+
+One first launches Ginga and starts the RC plugin.
+This can be done from the command line::
+
+    ginga --modules=RC
+
+From within Python, connect with a RemoteClient object as
+follows::
+
+    from ginga.util import grc
+    host='localhost'
+    port=9000
+    viewer = grc.RemoteClient(host, port)
+
+This viewer object is now linked to the Ginga using RC.
+
+Load an Image
+-------------
+
+One can load an image from memory in a channel of
+one's choosing.  First connect to a Channel::
+
+    ch = viewer.channel('Image')
+
+Then load a numpy image (i.e. any 2D ndarray)::
+
+    img = np.ones((100,100))
+    ch.load_np('any_str_here', img, 'fits', {})
+
+The image will display in Ginga and can be manipulated
+as usual.
+
+Overlay a Canvas Object
+-----------------------
+
+It is possible to add objects to the Canvas in a given
+Channel.  First connect::
+
+    canvas = viewer.canvas('Image')
+
+This connects to the Channel named "Image".  One can
+clear the objects drawn in the Canvas::
+
+    canvas.clear()
```

Comment:
Ok, figured it out. My clear now attempts to only remove
the added objects and not the image (which is removed by
delete_all_objects).  If you have something more elegant, please suggest it.